src/components/MetadataForm/MetadataFormStyles.ts

```diff
@@ -1,12 +1,12 @@
 import styled from 'styled-components';
-import { ComponentProps } from 'react';
+import { ComponentProps, FC } from 'react';
 import { Heading, themeSpacing } from '@amsterdam/asc-ui';
 
 export const MetadataFormStyle = styled.form`
 	display: flex;
 	flex-direction: column;
 `;
 
-export const StyledHeading = styled(Heading)<ComponentProps<typeof Heading>>`
+export const StyledHeading: FC<ComponentProps<typeof Heading>> = styled(Heading)`
```

Comment:
Should we type on React.FC ? or React.FunctionComponent

Reply:
I wasn't able to get rid of the TS error after upgrading to the latest styled-components when overriding an already styled component in another way :man_shrugging: